index.js

```diff
@@ -138,3 +138,54 @@ newSightingForm.addEventListener('submit', function (event) {
 /************************** EVENTS PART 2 JS MINI CHALLENGE ******************************/
 
 
+//**********Deliverable 1: Like an animal sighting**********
+
+const animSightings = document.querySelector('ul#animals')
```

Comment:
I would be a bit more descriptive with this variable.  `animalSightingsUL` I usually like to include the type of element it is if the variable's value is an HTML element.